packages/local_db/seed.sample.json

```diff
@@ -127,47 +127,135 @@
     }
   },
   "grants": {
-    "fWmHtS89edfHhVwOOYOz": {
-      "askAmount": 0.5,
-      "builder": "0x55b9CB0bCf56057010b9c471e7D42d60e1111EEa",
-      "description": "This is long description for 1st grant",
-      "status": "completed",
-      "timestamp": 1660983774000,
-      "title": "1s Test grant"
-    },
-    "eOp1CMm83N1LzlVhcLZu": {
-      "askAmount": 0.5,
-      "builder": "0x60583563D5879C2E59973E5718c7DE2147971807",
-      "title": "2nd Test grant",
-      "description": "This is long description",
-      "status": "completed",
-      "timestamp": 1660993774000
-    },
-    "5nwkdmFIAecWO2XUyyWP": {
-      "askAmount": 0.1,
-      "builder": "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
-      "title": "3rd Test grant",
-      "description": "This is long description for 3rd grant",
-      "status": "completed",
-      "timestamp": 1660983774000
-    },
-    "9ZoLxv5ay18mnGn6BwqG": {
-      "askAmount": 0.25,
-      "builder": "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
-      "title": "4th Test grant",
-      "description": "This is long description for 3rd grant",
-      "status": "proposed",
-      "timestamp": 1660983774000
+        "g9EmnJROMeqydxP1BGSx": {
+            "askAmount": 0.125,
+            "builder": "0x55fF7E28e7cd43C104dC89Aa69285E0E6EBa064e",
+            "title": "Buidl Guild English Auction",
+            "description": "Buidl Guild is a testnet english auction platform where users can create an auction on NFTs.",
```

Comment:
"Buidl Guild is a testnet"... something seems of in this desc haha

Reply:
Cleaned it a bit 😅